### DISCUSSION TITLE
XCode build failing

### DISCUSSION THREAD

**Discussion by naiduGrit** [pointfreeco/swift-composable-architecture#1759] [Q&A]
I was able to build the project till yesterday. 
I don't know what was changed in between, the XCode build got failing with `import` issues.
I can see that the methods of Darwin got some issues.

<img width="1047" alt="Screenshot 2022-12-20 at 8 37 05 AM" src="https://user-images.githubusercontent.com/88182287/208573164-27a24377-46f9-4b0b-aee2-109108f20d44.png">

Did any of you face this issue? Can someone please help me fix this?

TCA Version: `0.20.0`
XCode: `14.0`
OS version: `Monetory 12.6`

**Comment by mbrandonw** [comment]
Hi @naiduGrit, what version of combine-schedulers is your project using?

Version [0.7.3](https://github.com/pointfreeco/combine-schedulers/releases/0.7.3) had a breaking change, so if you are using 0.7.3 or later you may need to pin to an earlier version.

Also, 0.20.0 of TCA is _really_ old. You may want to consider upgrading to a newer version.

**Reply by naiduGrit** [reply]
Hi @mbrandonw
I have combine-schedulers version: `0.9.1`.

Yes, I can update. Wanted to know why this failing suddenly.
